### DISCUSSION TITLE
Is it possible to remove TimezoneData and icudt from the release distros?

### DISCUSSION THREAD

**Discussion by elringus** [unoplatform/Uno.Wasm.Bootstrap#438] [Q&A] [answered]
Those two files are 4MB in total, while the total average build size is around 9MB. Are they critical for the runtime?

![](https://i.gyazo.com/a687e94764e2b23c0b3f2b76e810c980.png)

**Comment by jeromelaban** [comment, marked as answer]
Not at this point, there's no configuration for it. TimezoneData.clr could be removed, but the runtime fails without `icudt.dat`. Those files should be compressing pretty well with brotli, though.
